.prettierignore

```diff
@@ -0,0 +1,2 @@
+src/bindings
+.gitignore
```

Comment:
I find it pretty hard to keep prettierignore / gitignore synchronized correctly in projects. If possible it would be nice to continue using gitignore. (Maybe by running prettier after generating bindings.)